annubes/tasks/task.py

```diff
@@ -11,15 +11,10 @@
 
 
 @dataclass
-class TaskSettings:
-    """Class for defining a task's settings.
+class TaskSettingsMixin:
+    """Mixin data class for defining attributes related to extra Task settings.
```

Comment:
Maybe the docstring could use a bit more explanation of why some are here and others are there. Maybe something along the lines of this:

```suggestion
class TaskSettingsMixin:
    """Mixin data class defining detailed parameters for `Task`.
    
    These settings are expected to be maintained throughout most experiments, whereas the attributes of `Task` itself are expected to be more commonly adjusted between individual experiments.
```